<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowRecuperar</class>
 <widget class="QMainWindow" name="WindowRecuperar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(239, 239, 239);
background-color: rgb(54, 54, 80);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TxtRecPass">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>80</y>
      <width>431</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Shonar Bangla</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Shonar Bangla&quot;;
color: rgb(151, 210, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Recuperar senha &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRecNo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>421</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Atençao E-mail nao cadastrado, crie uma conta !&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRecOk">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(174, 255, 225);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;As intruções para a recuperação &lt;br/&gt;serão enviadas para seu E-mail.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>167</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Email :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGetEmail">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>164</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    
	color: rgb(56, 56, 83);
	border-radius:3px;
	color:rgb(23,25,25);
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover{
background-color:rgb(170,170,255);
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecSair">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ 
	border-radius:3px;
	color: rgb(54, 54, 81);
	background-color: rgb(223, 223, 223);
}

QPushButton:hover{
	
	
	color: rgb(54, 54, 81);
	background-color: rgb(255, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>275</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 300 8pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;                    N O C z  @ All CopyRight   - 2019&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnviar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ 
	border-radius:3px;
	color: rgb(54, 54, 81);
	background-color: rgb(223, 223, 223);
}

QPushButton:hover{
	
	
	color: rgb(54, 54, 81);
	background-color: rgb(255, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>220</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>RomanC</family>
      <pointsize>28</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Acesso ao sistema &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(112, 193, 255);
font: 300 18pt &quot;Segoe UI&quot;;
font: 28pt &quot;RomanC&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;N O C z&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>lblRecOk</zorder>
   <zorder>lblRecNo</zorder>
   <zorder>label_2</zorder>
   <zorder>txtGetEmail</zorder>
   <zorder>btnRecSair</zorder>
   <zorder>label_3</zorder>
   <zorder>btnEnviar</zorder>
   <zorder>label</zorder>
   <zorder>TxtRecPass</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnRecSair</sender>
   <signal>clicked()</signal>
   <receiver>WindowRecuperar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
